src/Avalonia.Controls/ContentControl.cs

```diff
@@ -43,6 +43,23 @@ public class ContentControl : TemplatedControl, IContentControl, IContentPresent
         static ContentControl()
         {
             ContentProperty.Changed.AddClassHandler<ContentControl>((x, e) => x.ContentChanged(e));
+            TemplateProperty.OverrideDefaultValue<ContentControl>(new FuncControlTemplate((c, ns) =>
+            {
+                var presenter = new ContentPresenter
+                {
+                    [~BackgroundProperty] = c.GetObservable(BackgroundProperty).ToBinding(),
```

Comment:
I'm not sure if this will be a problem, but I think this means that the subscriptions to the `ContentControl` properties will never be released. So if the default value is used, and then a template is supplied, the original default template subscriptions won't be removed.

Reply:
I think we should probably use `TemplateBinding`s here to fix this.

Reply:
`TemplateBinding` has now been moved to Avalonia.Base by #11483.